### PR TITLE
Fix Issue7242 iOS FontSize is calculated incorrectly

### LINKED ISSUE #7242: [Bug] iOS FontSize is calculated incorrectly
### Description
On iOS named fontsizes are calculated incorrectly usually resulting in 1 point less than expected.
### Steps to Reproduce
1.  Executing 
` var result = new FontSizeConverter().ConvertFromInvariantString(nameof(NamedSize.Micro));`
or add a label with fontsize micro

### Expected Behavior
NamedSize.Micro should return 12 with default settings
### Actual Behavior
Returns 11
### Basic Information
I think the issue lies in [https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.iOS/Forms.cs](url)

The font scaling factor is calculated on line 188 like this.
`_fontScalingFactor = (double)UIFont.PreferredBody.PointSize / 18f;`
I think it should be :
`_fontScalingFactor = (double)UIFont.PreferredBody.PointSize / 17f;`

Why?
The default UIFont.PreferredBody.PointSize is 17 on my emulator iPhone 7 iOS 12.2.
Also on the Apple guidelines [https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/](url)
It states that the default body Size in points is 17.

And on line 224 in the source it is also stated that
`// iOS docs say default button font size is 15, default label font size is 17 so we use those as the defaults.`

- Version with issue: Xamarin.Forms 4.1.0.618606
- Last known good version: Xamarin.Forms 4.0.0.425677
- IDE: Visual Studio 2019
- Platform Target Frameworks: 
  - iOS:  8.0
- Nuget Packages: Xamarin.Forms
- Affected Devices: only tested on iPhone 7 iOS 12.2

### PR DESCRIPTION
**Pull request by Omniq-APL-Rollving**
### Description of Change ###
Changed the _fontScalingFactor so that the font size values corresponds with apples human interface guidelines (https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/). 

The change was made in file C:\…\Xamarin.Forms\Xamarin.Forms.Platform.iOS\Forms.cs, line 199 was changed from:
“_fontScalingFactor = (double)UIFont.PreferredBody.PointSize / 18f;”
to:
“_fontScalingFactor = (double)UIFont.PreferredBody.PointSize / 17f;”.
The issue was first introduced when scaling was added 
https://github.com/xamarin/Xamarin.Forms/pull/5985.

### Issues Resolved ### 
- fixes #7242

### API Changes ###
 None

### Platforms Affected ### 
- iOS

### Behavioral/Visual Changes ###
Text sizes will increase slightly because of the slight bump in font size for example micro that changes from 11 to 12 and medium/default goes from 16 to 17 which is apples recommended body text size.

### Before/After Screenshots ### 
Before: iPhone 11 iOS 13 3 2020-03-03 14_34_53

After: iPhone 11 iOS 13 3 2020-03-03 14_29_11

### Testing Procedure ###
I was unable to test the changes in any other way than testing manually checking values from the issue case and comparing results to apples guidelines.

### PR Checklist ###
- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by AndreiMisiukevich** [comment]
Hey, I assume you selected the wrong target branch :)

**Comment by Omniq-APL-Rollving** [comment]
Oh, I'm sorry I might have misunderstood the instructions what branch should I have targeted?
@AndreiMisiukevich

**Comment by AndreiMisiukevich** [comment]
> Oh, I'm sorry I might have misunderstood the instructions what branch should I have targeted?
> @AndreiMisiukevich

what branch did you select as base for your PR?

**Comment by Omniq-APL-Rollving** [comment]
Thanks for the heads up, I missed that I pulled version 4.6 and not  4.5

File: Xamarin.Forms.Platform.iOS/Forms.cs [hunk 1]
```diff
@@ -196,7 +196,7 @@ public IOSPlatformServices()
 #if __MOBILE__
 				//The standard accisibility size for a font is 18, we can get a
```
Comment (VladislavAntonyuk) [inline Xamarin.Forms.Platform.iOS/Forms.cs]: Comment also need to be updated

File: Xamarin.Forms.Platform.iOS/Forms.cs [hunk 2]
```diff
@@ -194,9 +194,9 @@ class IOSPlatformServices : IPlatformServices
 			public IOSPlatformServices()
 			{
 #if __MOBILE__
-				//The standard accisibility size for a font is 18, we can get a
+				//The standard accisibility size for a font is 17, we can get a
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Platform.iOS/Forms.cs]: fix the typo while you're at it...

is there a reference doc somewhere for this ? if so, please add a comment

**Comment by jfversluis** [comment]
@Omniq-APL-Rollving if you can address the last comment to fix the typo and add a relevant doc then I think we're all good to go here

**Pull request merged**